code/modules/mob/living/carbon/human/species_types/squidpeople.dm

```diff
@@ -19,19 +30,48 @@
 /datum/species/squid/random_name(gender,unique,lastname)
 	if(unique)
 		return random_unique_squid_name(genderToFind = gender)
-
 	var/randname = squid_name(gender)
-
 	if(lastname)
 		randname += " [lastname]"
-
 	return randname
 
 /proc/random_unique_squid_name(attempts_to_find_unique_name=10, genderToFind)
     for(var/i in 1 to attempts_to_find_unique_name)
         . = capitalize(squid_name(genderToFind))
         if(!findname(.))
             break
-			
+
 /datum/species/squid/after_equip_job(datum/job/J, mob/living/carbon/human/H)
-	H.grant_language(/datum/language/rlyehian)	
+	H.grant_language(/datum/language/rlyehian)
+	var/datum/action/innate/squid_change/S = new
+	S.Grant(H)
+
+/datum/species/squid/on_species_loss(mob/living/carbon/human/H)
+	fixed_mut_color = rgb(128,128,128)
+	H.update_body()
+
+/datum/action/innate/squid_change
+	name = "Color Change"
+	check_flags = AB_CHECK_CONSCIOUS
+	icon_icon = 'icons/mob/actions.dmi'
+	button_icon_state = "squid"
+	var/cooldown = 0
+
+/datum/action/innate/squid_change/IsAvailable()
+    if(cooldown > world.time)
+        return FALSE
+    return ..()
+
+/datum/action/innate/squid_change/Activate()
+	var/mob/living/carbon/human/H = owner
+	H.dna.species.fixed_mut_color = pick("FFA500", "B19CD9", "ADD8E6", "551A8B", "0000FF",
+    "32CD32", "D3D3D3", "2956B2", "FF0000", "00FF00", "FF69B4", "FFD700", "F58B57", "5AC18E",
```

Comment:
These could probably be in a static list for the species type.

Reply:
Throws an undefined var if I move the elsewhere, why even use one, I thought